.github/workflows/ci.yml

```diff
@@ -461,7 +461,7 @@ jobs:
       - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
       - uses: ./.github/actions/setup-shell
         with:
-          nix-shell: 'ci'
+          nix-shell: 'ci-cross' # Need cross-compiler for ASM simplification
```

Comment:
Did you forget to copy the `nix-cache: 'true'` here? Or was that intentionaly omited?